e2e/playwright-ci.config.js

```diff
@@ -7,13 +7,13 @@ const { devices } = require('@playwright/test');
 
 /** @type {import('@playwright/test').PlaywrightTestConfig} */
 const config = {
-    retries: 1,
+    retries: 3,
```

Comment:
Because we are erroring out on five failed tests per shard.

Effectively if the same test runs four times and succeeds on the 4th, the overall suite will pass

Reply:
can we put this in a comment please for future developers?